src/users/repository/userRepository.ts

```diff
@@ -16,11 +16,13 @@ export class UserRepository {
   }
 
   findAll(): Promise<User[]> {
-    return this.userRepository.find({relations: ["events"],});
+    return this.userRepository.find({ relations: ["pivot", "pivot.event"] });
```

Comment:
I don't we we need findAll without where parameter, also relations should optional